practices/smart_goals.md

```diff
@@ -0,0 +1,123 @@
+
+### Specific
+
+Specify your goal to be able to focus your efforts. The following questions can help you: 
+
+- What do I want to accomplish?
+- Why is this goal important?
+- Who is involved?
+- Where is it located?
+- Which resources or limits are involved?
+
+### Measurable
+
+With measurable goals set, you can track your progress and stay motivated. Clearly set deadlines will help you to assess your progress. 
+A measurable goal should address questions such as:
+
+- How much?
+- How many?
+- How will I know when it is accomplished?
+
+### Attainable
+
+Your goal also needs to be realistic and attainable to be successful. When you set an achievable goal, you may be able to identify previously overlooked opportunities or resources that can bring you closer to it.
+To set an attainable goal, you will have to deal with the following questions:
+
+- How can I accomplish this goal?
+- How realistic is the goal, based on other constraints, such as financial factors?
+
+**NOTE**: Do not set goals that depend on someone else. Only goals that depend on you, your effort, and your skills are convenient ones. 
+
+### Relevant
+
+Ensure that your goal matters to you, and is not in conflict with your other relevant goals. So, make sure that your plans drive everyone forward, but that you're still responsible for achieving your own goal.
+In order to set a relevant goal, answer these questions (answers should be “Yes”): 
+
+- Does this seem worthwhile?
+- Is this the right time?
+- Does this match my other efforts?
+- Am I the right person to reach this goal?
+- Is it applicable in the current (socio-)economic environment?
+
+### Time-bound
+
+What is your goal time-frame? An end-date can provide motivation and help you to prioritize and prevent everyday tasks from taking priority over your longer-term goals.
+To set a time-bound goal, you will deal with the following questions:
+
+- When?
+- What can I do six months from now?
+- What can I do six weeks from now?
+- What can I do today?
+
+## How to Start With SMART Goals
+
+There are various SMART Goals worksheets, templates, and maps that can help you with the goals setting process. 
+
+![](/files/smart_templ1b.png)
+
+## Problems That SMART Goals Can Solve
+
+SMART Goals can be used for problems that you are able to define fully following SMART Goals methodology carefully. 
+
+There are various goals that SMART Goals cannot solve: 
+
+- goals that do not allow definition of a clear result
+- goals whose progress cannot be measured
+- goals whose achieving depends on circumstances that you cannot affect
+- etc.
+
+For example, it will be very difficult (or impossible) to define a clear result and relevant metrics for problems like __**Not my problem mentality**__.
+
+## SMART Goals Pitfalls 
+
+Significant problems of SMART goal setting:
+
+- Focus is too narrow – fixating on a single SMART goal without the appropriate context, it is easy to fall into the trap of seeing it as the only goal.
```

Comment:
I like this style. It is something I struggle with since most of the other articles are showing pitfalls as a single point (The sales team poorly sets priorities). 

On the other hand, bloated points don't look pleasing. Where would you stand on this matter?

Reply:
I do not know why this happens. I use still the same syntax for list items. :o

Reply:
OK, I have converted the bulleted list with long items into sub-chapters. Looks better.